scripts/boxgame.js

```diff
@@ -0,0 +1,119 @@
+class BoxGameComponent {
+    box = {};
```

Comment:
why are these class properties being declared outside of a constructor without this.?
using field declarations like this is an experimental feature and wouldnt typically be used as production code without transpiling first. see https://github.com/tc39/proposal-class-fields
this code will break in many browsers because it has not been transpiled. So it fails in Safari